docs/source_files/pixi_workspaces/minijinja/task_args/pixi.lock

```diff
@@ -0,0 +1,7 @@
+version: 6
```

Comment:
What is the point of this lock file? I rather keep the folder similar to before. Where they are just named files in the folder.

Reply:
We lock all examples in `pixi_workspaces`. That way, we ensure that our tests stay reproducible.

Reply:
ah okay, really shows that we should get rid of writing empty lockfiles.